### PR TITLE
Use CMake-defined installation dir instead of hardcoded

### LINKED ISSUE #506: Install libs in proper /usr/lib64 destination
Problem: now openxray installs libs in `/usr/lib` but there is few issues because of this so is it possible to install them into `/usr/lib64`? How to setup this during build process?

### LINKED ISSUE #589: gentoo: Error while loading shared libraries
Hi, 
after compile the project in gentoo linux, I copied the root game files from wine location   ~/.wine/drive_c/Program Files/GSC Game World/S.T.A.L.K.E.R. - Call of Pripyat/   to   ~/.local/share/GSC/SCOP/

but, if I try to start the engine, I get the next output:

```
$ /usr/games/xr_3da 
/usr/games/xr_3da: error while loading shared libraries: xrEngine.so: cannot open shared object file: No such file or director
```

There is no additional description over what could be and I have not idea what happens, 
please, somebody so kind to help me?

### PR DESCRIPTION
**Pull request by 007exe**
Cmake edits can now use -DCMAKE_INSTALL_PREFIX =/usr and -DCMAKE_INSTALL_LIBDIR=lib or lib64 to indicate the location of libraries, this will solve compatibility problems with non-ubuntu-like distributions such as CentOS, Fedora, ALT and others. For Ubuntu, it is recommended to use -DCMAKE_INSTALL_BINDIR=/usr/games.

### CODE REVIEW

File: CMakeLists.txt [hunk 1]
```diff
@@ -228,3 +232,13 @@ if (UNIX)
         add_custom_target(uninstall COMMAND ${CMAKE_COMMAND} -P ${CMAKE_CURRENT_BINARY_DIR}/cmake_uninstall.cmake)
     endif()
 endif()
+
+
```
Comment (Xottab-DUTY) [inline CMakeLists.txt]: Redundant blank line.
```suggestion
```

File: src/xr_3da/CMakeLists.txt [hunk 2]
```diff
@@ -19,7 +19,8 @@ add_executable(${PROJECT_NAME} ${${PROJECT_NAME}__SOURCES} ${${PROJECT_NAME}__IN
 set_target_properties(${PROJECT_NAME} PROPERTIES PREFIX "")
 target_link_libraries(${PROJECT_NAME} xrCore xrAPI xrEngine)
 
-install(TARGETS ${PROJECT_NAME} RUNTIME DESTINATION /usr/games PERMISSIONS OWNER_READ OWNER_WRITE OWNER_EXECUTE
+# ${CMAKE_INSTALL_BINDIR}
```
Comment (Xottab-DUTY) [inline src/xr_3da/CMakeLists.txt]: Redundant comment
```suggestion
```

Comment (Xottab-DUTY) [reply src/xr_3da/CMakeLists.txt]: Also, is it possible to specify to CMake that we are installing game?

*(возможно ли как сказать симейку, что мы не простой бинарь устанавливаем, а игру?)*

File: CMakeLists.txt [hunk 3]
```diff
@@ -1,6 +1,11 @@
 cmake_minimum_required(VERSION 2.8.0)
+#Redirecting the default installation path /usr/local to /usr no need to use -DCMAKE_INSTALL_PREFIX =/usr
+set(CMAKE_INSTALL_PREFIX /usr)
 project(OpenXRay)
 
+#installation paths
```
Comment (Xottab-DUTY) [inline CMakeLists.txt]:
```suggestion
# Installation paths
```

File: CMakeLists.txt [hunk 4]
```diff
@@ -1,6 +1,11 @@
 cmake_minimum_required(VERSION 2.8.0)
+#Redirecting the default installation path /usr/local to /usr no need to use -DCMAKE_INSTALL_PREFIX =/usr
```
Comment (Xottab-DUTY) [inline CMakeLists.txt]:
```suggestion
# Redirecting the default installation path /usr/local to /usr no need to use -DCMAKE_INSTALL_PREFIX =/usr
```

**Review by Xottab-DUTY — APPROVED** [review]

**Review by eagleivg — APPROVED** [review]
Looks good enough.

**Pull request merged**